.travis.yml

```diff
@@ -49,14 +49,16 @@ env:
 
 install:
     - python -c "$(curl -fsSL https://raw.githubusercontent.com/platformio/platformio/master/scripts/get-platformio.py)"
-    - wget https://github.com/xxxajk/spi4teensy3/archive/master.zip -O /tmp/spi4teensy3.zip
-    - wget https://github.com/xxxajk/xmem2/archive/master.zip -O /tmp/xmem2.zip
-    - wget https://github.com/xxxajk/generic_storage/archive/master.zip -O /tmp/generic_storage.zip
-    - wget https://github.com/xxxajk/RTClib/archive/master.zip -O /tmp/RTClib.zip
-    - unzip /tmp/spi4teensy3.zip -d /tmp
-    - unzip /tmp/xmem2.zip -d /tmp
-    - unzip /tmp/generic_storage.zip -d /tmp
-    - unzip /tmp/RTClib.zip -d /tmp
+
+    # Use PlatformIO from development branch
+    # @TODO remove this line after 2.0.1 release
+    - pip install https://github.com/platformio/platformio/archive/develop.zip
+
+    # Libraries from PlatformIO Library Register
+    # http://platformio.org/#!/lib/show/416/TinyGPS
```

Comment:
This library for ``PLATFORMIO_CI_SRC=examples/pl2303/pl2303_gps/pl2303_gps.ino`` example.